addons/binding/org.openhab.binding.regoheatpump/src/main/java/org/openhab/binding/regoheatpump/handler/Rego6xxHeatPumpHandler.java

```diff
@@ -0,0 +1,405 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
Please note that I already run the above mentioned command (actually just re-run it and got no changes).